server/svix-server/src/v1/endpoints/endpoint/crud.rs

```diff
@@ -58,9 +60,11 @@ pub(super) async fn list_endpoints(
     })
     .collect();
 
-    Ok(Json(EndpointOut::list_response_no_prev(
+    Ok(Json(EndpointOut::list_response_asc(
         results,
         limit as usize,
+        is_prev,
+        newest_first.unwrap_or(false),
```

Comment:
Why not set a default in the newest_first structure directly and have it non-null?

Reply:
This way the struct remains more generic and can be reused elsewhere where default `true` is needed, and number of lines-wise it ends up being more or less the same. But I don't feel super strong about this, if you think having a default on the struct itself is significantly better I'm ok with changing to that.

Reply:
Resolving this since it's not a boolean anymore, therefore not relevant.